lib/dspace-schemas.ts

```diff
@@ -0,0 +1,98 @@
+import { z } from 'zod';
```

Comment:
This file is a WIP but is necessary to pull into this commit because it defines schemas related to the RSS feed. I'd really like to just merge this in and know that I'm still working on it on my other branch.